scripts/intersect_tool_yaml.py

```diff
@@ -38,7 +38,15 @@ def main():
             print("Processing: %s" % file)
         a = yaml.safe_load(open(file, 'r'))
         these_tools = a['tools']
+        tools_already_seen_on_this_instance = []
         for tool in these_tools:
+            # deal with tools that have the same name
+            tool_id = tool['name'] + tool['owner']
```

Comment:
```suggestion
            tool_id = '%s/%s' % (tool['name'], tool['owner'])
```

@natefoo / @Slugger70  are we py3 yet? Can I use the fancy `f"{tool['name']}/{tool['owner']}"` syntax already?

Reply:
This change should prevent issues like

```
alice/test
alicet/est
```

Reply:
Agree :+1: